backend/src/apiserver/model/experiment.go

```diff
@@ -15,12 +15,13 @@
 package model
 
 type Experiment struct {
-	UUID           string       `gorm:"column:UUID; not null; primary_key;"`
-	Name           string       `gorm:"column:Name; not null; unique_index:idx_name_namespace;"`
-	Description    string       `gorm:"column:Description; not null;"`
-	CreatedAtInSec int64        `gorm:"column:CreatedAtInSec; not null;"`
-	Namespace      string       `gorm:"column:Namespace; not null; unique_index:idx_name_namespace;"`
-	StorageState   StorageState `gorm:"column:StorageState; not null;"`
+	UUID                  string       `gorm:"column:UUID; not null; primary_key;"`
+	Name                  string       `gorm:"column:Name; not null; unique_index:idx_name_namespace;"`
+	Description           string       `gorm:"column:Description; not null;"`
+	CreatedAtInSec        int64        `gorm:"column:CreatedAtInSec; not null;"`
+	LastRunCreatedAtInSec int64        `gorm:"column:LastRunCreatedAtInSec; not null;"`
```

Comment:
@chensun fyi, this change adds a new column to Experiment table to allow for tracking for the last created run. 

Also note here: https://github.com/kubeflow/pipelines/pull/11163/files#diff-62d02d47cfac0eb11fdad7cf760bac31eb64f26e05aec490934c05bfd7949c09R569 which makes an additional db call during run creation calls.

Reply:
Adding new columns should be fine. Thanks for the callout.